version/version.go

```diff
@@ -1,4 +1,4 @@
 package version
 
 // VERSION ...
-const VERSION = "1.46.0"
+const VERSION = "1.47.0"
```

Comment:
please update the version in the tests as well

Reply:
Can you point me to the right direction? I just checked the same folder 😇

Reply:
most like that is the test that made this build failing

Reply:
anyways, I'm not sure if it makes sense to fix this, as we won't re-release this version, I guess :)

Reply:
Found it here:
https://github.com/gaborszakacs/bitrise/blob/patch-1/_tests/integration/version_test.go